server/dist/test/index.js

```diff
@@ -1,6 +1,224 @@
-import chaiHttp from 'chai-http';
-import chai from 'chai';
+'use strict';
```

Comment:
This is an autogenerated file/folder. It should not be here

Reply:
i added it to gitignore, i probably didn't do it well, i will check it